templates/hello/simple-snippet.html

```diff
@@ -0,0 +1,237 @@
+          _sendEvent('hideHighlight');
+        };
+
+        Mozilla.UITour.showMenu = function(name, callback) {
+          var showCallbackID;
+          if (callback)
+            showCallbackID = _waitForCallback(callback);
+
+          _sendEvent('showMenu', {
+            name: name,
+            showCallbackID: showCallbackID,
+          });
+        };
+
+        Mozilla.UITour.hideMenu = function(name) {
+          _sendEvent('hideMenu', {
+            name: name
+          });
+        };
+
+        Mozilla.UITour.getConfiguration = function(configName, callback) {
+          _sendEvent('getConfiguration', {
+            callbackID: _waitForCallback(callback),
+            configuration: configName,
+          });
+        };
+
+        // custom
+        Mozilla.UITour.isTargetAvailable = function(target, track) {
+          var ret = false;
+          Mozilla.UITour.getConfiguration('availableTargets', function (config) {
+            var targets = config.targets;
+            if (targets && targets.indexOf(target) > -1) {
+              if (track) sendMetric(ELIGIBLE_VIEW);
+              ret = true;
+            }else{
+              if (track) sendMetric(INELIGIBLE_VIEW);
+              ret = false;
+            }
+          });
+          return ret;
+        };
+
+        var bindHelloObserver = function() {
+          Mozilla.UITour.observe(function(e) {
+            switch (e) {
+              case 'Loop:ChatWindowOpened':
+                sendMetric(START_CONVERSATION);
+                break;
+              case 'Loop:RoomURLCopied':
+                sendMetric(URL_COPIED);
+                break;
+              // bug 1118346
+              case 'Loop:RoomURLEmailed':
+                sendMetric(URL_EMAILED);
+                break;
+            }
+          });
+        };
+
+       var handleVisibilityChange = function() {
+          if (document.hidden) {
+            // hide Hello menu & stop observer when changing tabs or minimizing window
+            Mozilla.UITour.observe(null);
+          } else {
+            // listen for Hello menu/chat window events
+            bindHelloObserver();
+          }
+        };
+
+        var thumbsUp = function() {
+          bindHelloObserver();
+          Mozilla.UITour.showMenu (TARGET_NAME);
+          // bug 1115227
+          Mozilla.UITour.registerPageID(PAGE_ID_SOURCE);
+          // bug 1113896
+          document.addEventListener('click', function(e) {
+              Mozilla.UITour.hideMenu (TARGET_NAME);
+          }, false);
+          document.addEventListener('visibilitychange', handleVisibilityChange);
+          sendMetric(ELIGIBLE_CLICK);
+        };
+
+        document.getElementById('hello-cta').addEventListener('click', function(e) {
+          e.preventDefault();
+          if (TARGET_AVAILABLE == true) thumbsUp()
+          else {
+            sendMetric(INELIGIBLE_CLICK);
+            window.location.replace('{{ icon_url }}');
```

Comment:
`window.location = '{{ icon_url }}'` should suffice here. Also make sure to test if you need `|safe` appended to the end of this for weird URLs or not.